packages/grid/x-data-grid-premium/src/tests/cellSelection.DataGridPremium.test.tsx

```diff
@@ -303,4 +310,82 @@ describe('<DataGridPremium /> - Cell Selection', () => {
       });
     });
   });
+
+  describe('Auto-scroll', () => {
```

Comment:
Use `clock.withFakeTimers();` in this section instead of enabling fake timers for the whole file?